scrapy/logformatter.py

```diff
@@ -43,6 +43,7 @@ def crawled(self, request, response, spider):
                 'request_flags' : request_flags,
                 'referer': referer_str(request),
                 'response_flags': response_flags,
+                'flags': response_flags
```

Comment:
Could you please add a comment which says that `flags` alias is for backwards compatibility with Scrapy < 1.4?